env/dev/user.clj

```diff
@@ -4,11 +4,34 @@
             [clojure.data.json :as json]
             [clojure.string :as str]
             [hawk.core :as hawk]
-            [clojure.tools.reader.edn :as edn])
+            [clojure.tools.reader.edn :as edn]
+            [clojure.tools.reader :as reader]
+            [clojure.tools.reader.reader-types :refer [string-push-back-reader]])
   (:import (java.net NetworkInterface InetAddress Inet4Address)
-           (java.io File)))
+           (java.io File PushbackReader)))
 ;; This namespace is loaded automatically by nREPL
 
+(defn- read-clj
+  "read a clojure file into a sequence of edn objects.
+
+  WARNING:
+   EDN is a subset of Clojure code representations so it is not guarantee
```

Comment:
guaranteed*

Reply:
Oh actually that description is no longer valid I have to update it 😅